resources/js/src/app/components/itemList/CategoryImageCarousel.js

```diff
@@ -86,7 +86,7 @@ Vue.component("category-image-carousel", {
                 },
                 onInitialized: event =>
                 {
-                    if (this.showNav === "true")
+                    if (this.showNav === true)
```

Comment:
Der Vergleich `=== true` ist überflüssig. Gibt es hier inzwischen eine Konvention in der Schreibweise?

Reply:
Naja die Konvention ist das man es nicht so macht. Habe irgendwie nicht nachgedacht wo ich das geändert habe.